src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoidnoargumentsuperconstructorcall/Example1.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidNoArgumentSuperConstructorCall"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.avoidnoargumentsuperconstructorcall;
+
+// xdoc section -- start
+class SuperClass {
+  public SuperClass() {}
+  public SuperClass(int arg) {}
+}
+class Example1 extends SuperClass {
+  Example1() {
+    super(); // violation
+  }
+
+  Example1(int arg) {
+    super(arg); // OK, call with argument have to be explicit
+  }
+
+  Example1(long arg) {
+    // OK, call is implicit
+  }
+}
```

Comment:
```suggestion
class Example1 extends SuperClass {
  Example1() {
    super(); // violation
  }

  Example1(int arg) {
    super(arg); // ok, explicit constructor invocation with argument
  }

  Example1(long arg) {
    // ok, no explicit constructor invocation
  }
}
```

Reply:
Consider: Checkstyle doesn't know (and shouldn't care) about what the compiler is doing for the most part. `OK, call is implicit` really doesn't make any sense in this context; this check only cares that there is no constructor invocation at all here,

Reply:
Complete, please review.

Reply:
@Lmh-java, please do not resolve review items, just reply to each of review items. Maintainers will resolve as confirmation that update was done as expected.

Reply:
Got it, sorry about that.